src/Senders/Jaeger.Senders.Grpc/README.md

```diff
@@ -0,0 +1,39 @@
+# gRPC sender for Jaeger (Jaeger.Senders.GRPC)
+
+## Usage
+This package contains the `GrpcSenderFactory` for use with `SenderResolver`.
+
+To use this, include the following line before doing any other calls to the `Configuration` API.
+
+```C#
+Configuration.SenderConfiguration.DefaultSenderResolver = new SenderResolver(loggerFactory)
+	// You can add other ISenderFactory instances too
+	.RegisterSenderFactory<ThriftSenderFactory>();
+```
+
+If you need to set `JAEGER_SENDER_FACTORY` (when multiple factories are registered), `GrpcSenderFactory.Name` corresponds to `"grpc"`.
+
+## Configuration
+Independent on what variables are set in `Configuration.SenderConfiguration`, a `GrpcSender` is created.
+
+```C#
+new SenderConfiguration(loggerFactory)
+	// Used with UdpSender:
```

Comment:
what does this mean?

Reply:
Wanted to replace it with GrpcSender. Copy-Paste error.